hw/ip/i2c/doc/registers.md

```diff
@@ -3,34 +3,35 @@
+| i2c.[`HOST_FIFO_STATUS`](#host_fifo_status)       | 0x2c     |        4 | Host mode FIFO status register                                                                     |
+| i2c.[`TARGET_FIFO_STATUS`](#target_fifo_status)   | 0x30     |        4 | Target mode FIFO status register                                                                   |
+| i2c.[`OVRD`](#ovrd)                               | 0x34     |        4 | I2C Override Control Register                                                                      |
+| i2c.[`VAL`](#val)                                 | 0x38     |        4 | Oversampled RX values                                                                              |
+| i2c.[`TIMING0`](#timing0)                         | 0x3c     |        4 | Detailed I2C Timings (directly corresponding to table 10 in the I2C Specification).                |
+| i2c.[`TIMING1`](#timing1)                         | 0x40     |        4 | Detailed I2C Timings (directly corresponding to table 10 in the I2C Specification).                |
+| i2c.[`TIMING2`](#timing2)                         | 0x44     |        4 | Detailed I2C Timings (directly corresponding to table 10 in the I2C Specification).                |
+| i2c.[`TIMING3`](#timing3)                         | 0x48     |        4 | Detailed I2C Timings (directly corresponding to table 10, in the I2C Specification).               |
+| i2c.[`TIMING4`](#timing4)                         | 0x4c     |        4 | Detailed I2C Timings (directly corresponding to table 10, in the I2C Specification).               |
+| i2c.[`TIMEOUT_CTRL`](#timeout_ctrl)               | 0x50     |        4 | I2C clock stretching timeout control.                                                              |
+| i2c.[`TARGET_ID`](#target_id)                     | 0x54     |        4 | I2C target address and mask pairs                                                                  |
+| i2c.[`ACQDATA`](#acqdata)                         | 0x58     |        4 | I2C target acquired data                                                                           |
+| i2c.[`TXDATA`](#txdata)                           | 0x5c     |        4 | I2C target transmit data                                                                           |
+| i2c.[`HOST_TIMEOUT_CTRL`](#host_timeout_ctrl)     | 0x60     |        4 | I2C host clock generation timeout value (in units of input clock frequency)                        |
+| i2c.[`TARGET_TIMEOUT_CTRL`](#target_timeout_ctrl) | 0x64     |        4 | I2C target internal stretching timeout control.                                                    |
+| i2c.[`TARGET_NACK_COUNT`](#target_nack_count)     | 0x68     |        4 | Number of times the I2C target has NACK'ed a new transaction since the last read of this register. |
```

Comment:
Can this commit be squashed into the previous commit (with commit messages merged), please?

Reply:
Yes :+1: